tests/net/socket/misc/src/main.c

```diff
@@ -286,6 +286,8 @@ void test_so_bindtodevice(int sock_c, int sock_s, struct sockaddr *peer_addr,
 	zassert_equal(ret, 0, "close failed, %d", errno);
```

Comment:
Please change the commit subject of "net: sockets: add tests for SO_REUSEPORT" to
`tests: net: sockets: add tests for SO_REUSEPORT`